up42/viztools.py

```diff
@@ -137,6 +138,12 @@ def plot_results(
                  e.g. matplotlib cmap etc.
         """
         warnings.filterwarnings("ignore", category=rasterio.errors.NotGeoreferencedWarning)
+        warn(
+            "Visualization methods are getting deprecated. The current feature will be discontinued "
+            "after March 31, 2024, and will be replaced by new processing functionalities.",
```

Comment:
If I am not wrong, we aren't providing any alternative for visualization in new processing functionalities right?
In this case, consider removing "and will be replaced by new processing functionalities."

Reply:
same for others below.

Reply:
I will add your suggestion thanks. I think adding new functionalities can be redundant.

But please be aware that this deprecation message will be exposed when the user runs this function from Job, JobCollection, and JobTask classes. I was referring to these classes when I added the new functionalities. Just to make sure we are all aware of the context.

Reply:
I see.
Maybe we can control that by updating the `stacklevel=2` param? 🤔